.github/workflows/reusable-ubuntu-ci.yml

```diff
@@ -0,0 +1,144 @@
+name: Fast-DDS Ubuntu CI reusable workflow
+
+on:
+  workflow_call:
+    inputs:
+      os-image:
+        description: 'The OS image for the workflow'
+        required: true
+        type: string
+      label:
+        description: 'ID associated to the workflow'
+        required: true
+        type: string
+      colcon-args:
+        description: 'Extra arguments for colcon cli'
+        required: false
+        type: string
+      cmake-args:
+        description: 'Extra arguments for cmake cli'
+        required: false
+        type: string
+      ctest-args:
+        description: 'Extra arguments for ctest cli'
+        required: false
+        type: string
+      fastdds-branch:
+        description: 'Branch or tag of Fast DDS repository (https://github.com/eProsima/Fast-DDS)'
+        required: true
+        type: string
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  reusable-ubuntu-ci:
+    runs-on: ${{ inputs.os-image }}
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-ci') }}
+    strategy:
+      fail-fast: false
+      matrix:
+        cmake-build-type:
+          - 'RelWithDebInfo'
+    steps:
+      - name: Sync eProsima/Fast-DDS repository
+        uses: eProsima/eProsima-CI/external/checkout@v0
+        with:
+          path: src/fastrtps
+          ref: ${{ inputs.fastdds-branch }}
+
+      - name: Get minimum supported version of CMake
+        uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: '3.22.6'
+
+      - name: Install apt dependencies
+        uses: eProsima/eProsima-CI/ubuntu/install_apt_packages@v0
+        with:
+          packages: libasio-dev libtinyxml2-dev libssl-dev
```

Comment:
I dont know whether we should specify the `libssl-dev` `1.1` version here

Reply:
I think we should, yes 
@EduPonz ?

Reply:
I don't think so. This workflow would run in ubuntu 20.04 which ship 1.1.1, so it's OK IMO